plugin.idea/src/com/microsoft/alm/plugin/idea/common/utils/VcsHelper.java

```diff
@@ -93,8 +95,13 @@ public static boolean isVstsRepo(final Project project) {
             if (!isGitVcs(project)) {
                 return false;
             }
-            final GitRepository repo = GitUtil.getRepositoryManager(project).getRepositoryForFile(project.getBaseDir());
-            return repo != null && TfGitHelper.isTfGitRepository(repo);
+
+            return ContainerUtil.exists(GitUtil.getRepositoryManager(project).getRepositories(), new Condition<GitRepository>() {
+                @Override
+                public boolean value(GitRepository gitRepository) {
+                    return TfGitHelper.isTfGitRepository(gitRepository);
```

Comment:
Do we need to check for null, like we previously did?

Reply:
@jeschu1  getRepositories() marked as NotNull and I think returned list shouldn't contain null elements. In this case, the null check not needed.